src/packer.json

```
@@ -46,7 +46,7 @@
         }
       },
       "tags": {
-        "Application": "Example",
+        "Application": "COOL",
```

We usually make the Application tag specific to the AMI:
* https://github.com/cisagov/assessor-portal-packer/blob/b8771840660e1fbb037f635e509e1541da580361/src/packer.json#L49
* https://github.com/cisagov/kali-packer/blob/1cd91902ad6f92146f67669c8b31d42ec3236f03/src/packer.json#L49
* https://github.com/cisagov/nessus-packer/blob/bb95dc457d11d410052ca477dbe208f4439c87b9/src/packer.json#L49
* https://github.com/cisagov/pca-gophish-composition-packer/blob/c1e30c7ec3aa5a4c9ad391f31f815644dd948ebe/src/packer.json#L49
```suggestion
        "Application": "Terraformer",
```